core/src/main/java/org/kiwix/kiwixmobile/core/base/BaseActivity.kt

```diff
@@ -47,7 +46,7 @@ abstract class BaseActivity : AppCompatActivity() {
 
   override fun setContentView(@LayoutRes layoutResID: Int) {
     super.setContentView(layoutResID)
-    unbinder = ButterKnife.bind(this)
+    // unbinder = ButterKnife.bind(this)
```

Comment:
why is this commented out? also we can remove the unbinder(but kep the `bind` call), it is only needed for Fragments and not activities